plugins/@grouparoo/demo/src/util/runAction.ts

```diff
@@ -1,6 +1,3 @@
-//import { Connection, Action, config, log, utils, api, action } from 'actionhero';
-//import * as dotProp from "dot-prop";
```

Comment:
it looks like this `dotProp` thing was being used in the commented out code below - should all that commented code be removed too?

Reply:
I think I'll leave it for now... not really sure what @bleonard was up to here :D